btree/btree.c

```diff
@@ -3085,6 +3085,594 @@ static void ff_rec_del_credit(const struct nd *node, m0_bcount_t ksize,
  *  --------------------------------------------
  */
 
+#if 0
+/**
+ *  --------------------------------------------------------
+ *  Section START -
+ *  Fix Sized Keys and Variable Sized Value Node Structure
+ *  ---------------------------------------------------------
+ */
+
+/**
+ *
+ * Proposed Node Structure for Fix Sized Keys and Variable Sized Value :
+ *
+ * +-----------+----+-+----+-+----+-+------------------------------------+----+----+----+
+ * |           |    | |    | |    | |                                    |    |    |    |
+ * |           |    | |    | |    | |                                    |    |    |    |
+ * |Node Header| K0 | | K1 | | K2 | | ----->                      <----- | V0 | V1 | V2 |
```

Comment:
The Values will be in the reverse order from the way it is shown here just like how Yash has it in his design.

Reply:
Yes right, it will be v2 ,v1 , v0, it is my mistake